### PR TITLE
EAS-1971

### PR DESCRIPTION
**Pull request by aliner-wang**
Scans for imports using require added.

### CODE REVIEW

File: crates/forge_analyzer/src/definitions.rs [hunk 1]
```diff
@@ -2896,7 +2948,14 @@ impl Visit for ImportCollector<'_> {
     }
 
     fn visit_import_star_as_specifier(&mut self, n: &ImportStarAsSpecifier) {
+        debug!("Does this ever get run?");
         let local = n.local.to_id();
+        // let defkind1 = self
```
Comment (jwong101) [inline crates/forge_analyzer/src/definitions.rs:2953]: Can you remove this comment?

File: crates/forge_analyzer/src/definitions.rs [hunk 2]
```diff
@@ -2294,37 +2296,83 @@ impl Visit for FunctionCollector<'_> {
...
-                            operand_stack: vec![],
-                            in_lhs: false,
+                        match args.get(0) {
+                            Some(ExprOrSpread { spread, expr: lit }) => {
+                                if let Expr::Lit(Lit::Str(Str { value, .. })) = &**lit {
+                                    let package_to_check = self
+                                        .secret_packages
+                                        .iter()
+                                        .find(|package| package.package_name == &**value);
+
```
Comment (jwong101) [inline crates/forge_analyzer/src/definitions.rs:2311]: Can you add a `TODO` to handle local module imports?

**Review by jwong101 — CHANGES_REQUESTED** [review]
Just a couple of nits, but overall looks good. Also, don't forget to sign the CLA.

File: crates/forge_analyzer/src/definitions.rs (hunk 1 above)
Comment (aliner-wang) [reply crates/forge_analyzer/src/definitions.rs]: Done

File: crates/forge_analyzer/src/definitions.rs (hunk 2 above)
Comment (aliner-wang) [reply crates/forge_analyzer/src/definitions.rs:2312]: Done

**Pull request closed without merging**